lib/portlayer/exec/handle.go

```diff
@@ -81,17 +80,15 @@ func newHandleKey() string {
 	return hex.EncodeToString(b)
 }
 
-func newHandle(con *Container) *Handle {
+func newHandle(con *Container, state State) *Handle {
```

Comment:
should not be taking a state parameter here - possible this shouldn't even be the same type as the State's we've got elsewhere.
The only valid states for this currently are:
StateStopped
StateRunning

We may want to add suspend/resume in the future, but this field is an imperative instruction, not a description and as such intermediate states will always be invalid here.